packages/format-library/src/text-color/index.js

```diff
@@ -17,6 +17,8 @@ import { removeFormat } from '@wordpress/rich-text';
  */
 import { default as InlineColorUI, getActiveColors } from './inline';
 
+export const transparentValue = 'rgba(0, 0, 0, 0)';
```

Comment:
I pushed a commit replacing all remaining occurrences of this string in `format-library` with the new constant. This should decrease the chances of format mismatches creeping in — for example, if the setter uses a hardcoded string like `rgba(0,0,0,0)` (with no spaces), the logic breaks.

The module constant, coupled with the expanded test coverage, might be enough. Otherwise we could use a regular expression.